connector/src/main/java/org/geysermc/connector/utils/LoginEncryptionUtils.java

```diff
@@ -155,12 +158,15 @@ private static void startEncryptionHandshake(GeyserSession session, PublicKey ke
     private static int AUTH_FORM_ID = 1336;
     private static int AUTH_DETAILS_FORM_ID = 1337;
 
-    public static void showLoginWindow(GeyserSession session) {
-        SimpleFormWindow window = new SimpleFormWindow("Login", "You need a Java Edition account to play on this server.");
-        window.getButtons().add(new FormButton("Login with Minecraft"));
-        window.getButtons().add(new FormButton("Disconnect"));
-
-        session.sendForm(window, AUTH_FORM_ID);
+    public static void showLoginWindow(GeyserSession session, AuthType mode) {
+    	if(mode == AuthType.SMART){
+            SimpleFormWindow window = new SimpleFormWindow("Login", "Please choose a way to join the server");
+            window.getButtons().add(new FormButton("Online"));
+            window.getButtons().add(new FormButton("Offline"));
+            window.getButtons().add(new FormButton("Disconnect"));
+            session.sendForm(window, AUTH_FORM_ID);
+        }
+        else showLoginDetailsWindow(session);
```

Comment:
The else should show the form we had before to warn the user the server is in online mode.

Reply:
That means I need to identify the authentication type of the server?

Reply:
No just show the warning form instead of straight to the login form like this does if its not SMART auth type